src/components/LoanFinding/QuickFiltersSection.vue

```diff
@@ -157,6 +157,9 @@ export default {
 			const params = new URLSearchParams(paramStr);
 			return `/lend/filter?${params.toString()}`;
 		},
+		// TODO: Rearchitect this at some point.
```

Comment:
I recommend adding a ticket to the backlog with some context on what is needed for this refactor.